tests/jballerina-unit-test/src/test/resources/test-src/services/service_decl.bal

```diff
@@ -179,7 +202,7 @@ function testServiceDecl() {
     reset();
 }
 
-function assertEquality(any|error actual, any|error expected) {
+function assertEquality(any|error expected, any|error actual) {
```

Comment:
Seems like some calls to this function still pass the actual as the first arg, shall we check and update?

Reply:
will do